.php-cs-fixer.dist.php

```diff
@@ -84,12 +84,11 @@
         ],
     ],
     'yoda_style' => false,
+    'php_unit_test_case_static_method_calls' => false,
```

Comment:
I would keep this rule, or shall we switch to:

```diff
- $this->assertSame(
+ self::assertSame(
```

Reply:
Agree, I like `$this->assertSame(` more too.